app/api/v2/handlers/operation_api.py

```diff
@@ -34,40 +34,80 @@ def add_routes(self, app: web.Application):
+                          }])
     @aiohttp_apispec.querystring_schema(BaseGetOneQuerySchema)
-    @aiohttp_apispec.response_schema(OperationSchema(partial=True))
+    @aiohttp_apispec.response_schema(OperationSchema(partial=True),
+                                     description='The response is the operation with the specified id, if any.')
     async def get_operation_by_id(self, request: web.Request):
         operation = await self.get_object(request)
         return web.json_response(operation)
 
     @aiohttp_apispec.docs(tags=['operations'],
-                          summary='Required nested schema fields are as follows: "adversary.adversary_id", '
-                                  '"planner.planner_id", and "source.id".')
-    @aiohttp_apispec.request_schema(OperationSchema())
-    @aiohttp_apispec.response_schema(OperationSchema)
+                          summary='Create a new CALDERA operation record',
+                          description='Create a new CALDERA operation using the format provided in the '
+                                      '`OperationSchema`. Required nested schema fields are as follows: '
+                                      '"adversary.adversary_id", "planner.planner_id", and "source.id"')
+    @aiohttp_apispec.request_schema(OperationSchema)
+    @aiohttp_apispec.response_schema(OperationSchema,
+                                     description='The response is the newly-created operation report.')
     async def create_operation(self, request: web.Request):
         operation = await self.create_object(request)
         return web.json_response(operation.display)
 
-    @aiohttp_apispec.docs(tags=['operations'])
-    @aiohttp_apispec.request_schema(OperationSchema(partial=True, only=['state',
-                                                                        'autonomous',
-                                                                        'obfuscator']))
-    @aiohttp_apispec.response_schema(OperationSchema(partial=True))
+    @aiohttp_apispec.docs(tags=['operations'],
+                          summary='Update fields within an operation',
+                          description='Update one CALDERA operation in memory based on the operation id (String '
+                                      'UUID). Any fields in the operation object may be edited in the request body '
+                                      'using the `OperationSchema`.',
+                          parameters=[{
+                              'in': 'path',
+                              'name': 'id',
+                              'schema': {'type': 'string'},
+                              'required': 'true',
+                              'description': 'UUID of the Operation object to be retrieved.'
+                          }])
+    @aiohttp_apispec.request_schema(OperationSchema(partial=True))
+    @aiohttp_apispec.response_schema(OperationSchema(partial=True, only=['state', 'autonomous', 'obfuscator']),
```

Comment:
Sorry one last tiny change - can you move `only=['state', 'autonomous', 'obfuscator']` into the `request_schema` for this endpoint instead? I think this was accidentally added to the `response_schema`